Index: src/task2/Rectangle.java
===================================================================
diff --git a/src/task2/Rectangle.java b/src/task2/Rectangle.java
deleted file mode 100644
--- a/src/task2/Rectangle.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,29 +0,0 @@
-package task2;
-
-public class Rectangle extends Figure{
-    private final int length;
-    private final int width;
-
-    public Rectangle(int length, int width) {
-        this.length = length;
-        this.width = width;
-    }
-
-    @Override
-    public double getPerimeter() {
-        return 2 * (length + width);
-    }
-
-    @Override
-    public double getSquare() {
-        return length * width;
-    }
-
-    @Override
-    public String toString() {
-        return "Type of the figure is circle" +
-                "\nIt's sides are: " + length + ", " + width +
-                "\nIt's perimeter = " + getPerimeter() +
-                "\nIt's square is " + getSquare();
-    }
-}
Index: src/task2/Triangle.java
===================================================================
diff --git a/src/task2/Triangle.java b/src/task2/Triangle.java
deleted file mode 100644
--- a/src/task2/Triangle.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,35 +0,0 @@
-package task2;
-
-public class Triangle extends Figure{
-    private final int firstSide;
-    private final int secondSide;
-    private final int thirdSide;
-
-    public Triangle(int firstSide, int secondSide, int thirdSide){
-        this.firstSide = firstSide;
-        this.secondSide = secondSide;
-        this.thirdSide = thirdSide;
-    }
-
-    @Override
-    public double getPerimeter() {
-        return firstSide + secondSide + thirdSide;
-    }
-
-    @Override
-    public double getSquare() {
-        double halfPerimeter = getPerimeter()/2;
-        return Math.sqrt(halfPerimeter *
-                (halfPerimeter - firstSide) *
-                (halfPerimeter - secondSide) *
-                (halfPerimeter - thirdSide));
-    }
-
-    @Override
-    public String toString() {
-        return "Type of the figure is triangle" +
-                "\nIt's sides are: " + firstSide + ", " + secondSide + ", " + thirdSide +
-                "\nIt's perimeter is " + getPerimeter() +
-                "\nIt's square is " + getSquare();
-    }
-}
Index: src/task2/Circle.java
===================================================================
diff --git a/src/task2/Circle.java b/src/task2/Circle.java
deleted file mode 100644
--- a/src/task2/Circle.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,27 +0,0 @@
-package task2;
-
-public class Circle extends Figure{
-    private final int radius;
-
-    public Circle(int radius){
-        this.radius = radius;
-    }
-
-    @Override
-    public double getPerimeter() {
-        return 2 * Math.PI * radius;
-    }
-
-    @Override
-    public double getSquare() {
-        return Math.PI * Math.pow(radius, 2);
-    }
-
-    @Override
-    public String toString() {
-        return "Type of the figure is circle" +
-                "\nIt's radius is " + radius +
-                "\nIt's perimeter = " + getPerimeter() +
-                "\nIt's square is " + getSquare();
-    }
-}
Index: src/task1/Accountant.java
===================================================================
diff --git a/src/task1/Accountant.java b/src/task1/Accountant.java
deleted file mode 100644
--- a/src/task1/Accountant.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,13 +0,0 @@
-package task1;
-
-public class Accountant extends Employee {
-
-    public Accountant(String name) {
-        super(name, "accountant");
-    }
-
-    @Override
-    public void printWork() {
-        System.out.println("I take calls from visitors");
-    }
-}
Index: src/MyMain.java
===================================================================
diff --git a/src/MyMain.java b/src/MyMain.java
deleted file mode 100644
--- a/src/MyMain.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,49 +0,0 @@
-import task1.*;
-import task2.*;
-
-public class MyMain {
-    public static void main(String[] args) {
-        //getEmployeeInfo(); //Task1
-        createFigures();//Task2
-    }
-
-    static void getEmployeeInfo(){
-        Employee[] office = new Employee[]{
-            new Director("Pavel"),
-            new Worker("Oleg"),
-            new Accountant("Anna")
-        };
-        for (Employee person: office){
-            person.printPosition();
-            person.printWork();
-            System.out.println("-------------------------");
-        }
-    }
-
-    static void createFigures(){
-        Figure[] figures = new Figure[]{
-            new Triangle(3, 4, 5),
-            new Rectangle(10, 5),
-            new Rectangle(14, 12),
-            new Circle(10),
-            new Circle(5)
-        };
-
-        for (Figure figure: figures){
-            System.out.println(figure);
-            System.out.println("----------------------------");
-        }
-
-        System.out.println("Sum of the figures' perimeters is " + getSumOfPerimeters(figures));
-    }
-
-    static double getSumOfPerimeters(Figure[] figures){
-        double sum = 0;
-        for (int counter = 0; counter < figures.length; counter++){
-            sum += figures[counter].getPerimeter();
-        }
-        return sum;
-    }
-}
-
-
Index: src/task1/Worker.java
===================================================================
diff --git a/src/task1/Worker.java b/src/task1/Worker.java
deleted file mode 100644
--- a/src/task1/Worker.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,13 +0,0 @@
-package task1;
-
-public class Worker extends Employee {
-
-    public Worker(String name) {
-        super(name, "worker");
-    }
-
-    @Override
-    public void printWork() {
-        System.out.println("I do the simple work the director assigns me");
-    }
-}
Index: src/task1/Printable.java
===================================================================
diff --git a/src/task1/Printable.java b/src/task1/Printable.java
deleted file mode 100644
--- a/src/task1/Printable.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,6 +0,0 @@
-package task1;
-
-public interface Printable {
-    void printPosition();
-    void printWork();
-}
Index: src/task1/Director.java
===================================================================
diff --git a/src/task1/Director.java b/src/task1/Director.java
deleted file mode 100644
--- a/src/task1/Director.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,13 +0,0 @@
-package task1;
-
-public class Director extends Employee {
-
-    public Director(String name) {
-        super(name, "director");
-    }
-
-    @Override
-    public void printWork() {
-        System.out.println("I'm a head of the office!");
-    }
-}
Index: src/task1/Employee.java
===================================================================
diff --git a/src/task1/Employee.java b/src/task1/Employee.java
deleted file mode 100644
--- a/src/task1/Employee.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,17 +0,0 @@
-package task1;
-
-public abstract class Employee implements Printable {
-    private final String name;
-    private final String position;
-
-    Employee(String name, String position){
-        this.name = name;
-        this.position = position;
-    }
-
-    @Override
-    public void printPosition() {
-            System.out.println("A position of the employee " + name + " is " + position);
-    }
-
-}
Index: src/task2/Figure.java
===================================================================
diff --git a/src/task2/Figure.java b/src/task2/Figure.java
deleted file mode 100644
--- a/src/task2/Figure.java	(revision 301d5955279516af347def2acc5c2f3400568d09)
+++ /dev/null	(revision 301d5955279516af347def2acc5c2f3400568d09)
@@ -1,6 +0,0 @@
-package task2;
-
-public abstract class Figure {
-    public abstract double getPerimeter();
-    public abstract double getSquare();
-}
